js/color-view.js

```diff
@@ -0,0 +1,56 @@
+import { html, render } from 'https://unpkg.com/lit-html@1.0.0/lit-html.js';
+
+class ColorPicker extends HTMLElement {
+    constructor() {
+        super();
+        this.render = this.render.bind(this);
+        this.setColor = this.setColor.bind(this);
+        this.applyColor = this.applyColor.bind(this);
+        this.onContextChanged = this.onContextChanged.bind(this);
```

Comment:
:neckbeard: You don't have to bind here if you write the methods as arrow funcs.

e.g. 
```
setColor = async (event) => {
        event.preventDefault();
        const color = this.querySelector('input').value;
        await fin.Platform.getCurrentSync().setWindowContext({ color });
    }
```

Doesn't matter functionality-wise for this one, but if part of our goal in demoing platforms is "look how little code it takes to make <thing> happen," it does cut out a few lines here and there.

Reply:
Agreed, I have another PR landing at some point this week with some cleanup (moving stuff to their own files etc) I will make the changes there.